src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/PackageReferenceProject.cs

```diff
@@ -25,6 +25,11 @@
 
 namespace NuGet.PackageManagement.VisualStudio
 {
+    internal static class PackageReferenceProject
```

Comment:
It would be great if we didn't have to have a new static class to store a global state of a telemetry event. Can we implement this in some other way?